examples/1_simple_runbooks/backup.py

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env python
+from brigade.easy import easy_brigade
+from brigade.plugins.tasks import files, networking, text
+
+
+def backup(task):
+    result = task.run(networking.napalm_get,
+                      getters="config")
+
+    return task.run(files.write,
+                    content=result.result["config"]["running"],
+                    filename="./backups/{}".format(task.host))
+
+
+brigade = easy_brigade(
```

Comment:
Using `brigade` name here is a confusing name choice since that is the name of the library also.

Reply:
you are right. Will fix that